emission/integrationTests/suggestionsys/category_of_business_nominatim.dataset.json

```diff
@@ -9,6 +9,127 @@
+        "input": {
+            "loc": {
+                "coordinates": [-118.4106739, 33.8842780],
+                "type": "Point"
+            }
+        },
+        "output": ["bakeries", "sandwiches", "coffee"]
+    },
+
+    {
+        "test_name": "Manhattan Beach FISHBAR", 
+        "test_type": "Suburban local bar/restaurant, non-chain",
+        "input": {
+            "loc": {
+                "coordinates": [-118.4182952, 33.9017119], 
+                "type": "Point"
+            }
+        },
+        "output": ["seafood", "breakfast_brunch", "newamerican"]
+    },
+
+    {
+        "test_name": "El Segundo Umi",
+        "test_type": "Sushi restaurant in small shopping center",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3947103, 33.9024449],
+                "type": "Point"
+            }
+        },
+        "output": ["japanese", "sushi", "seafood"]
+    },
+
+    {
+        "test_name": "Redondo Beach Captain Kidd's",
+        "test_type": "Fish market in seaside town",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3923475, 33.8434248],
+                "type": "Point"
+            }
+        },
+        "output": ["seafoodmarkets", "seafood", "fishnchips"]
+    },
+
+    {
+        "test_name": "Mira Costa High School Manhattan Beach",
+        "test_type": "Local public school in Los Angeles County",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3893806, 33.8729666],
+                "type": "Point"
+            }
+        },
+        "output": ["highschools"]
+    },
+
+    {
+        "test_name": "El Segundo High School",
+        "test_type": "Local public school in Los Angeles County",
+        "input": {
+            "loc": {
+                "coordinates": [-118.4141835, 33.9245280],
+                "type": "Point"
+            }
+        },
+        "output": ["highschools"]
+    },
+
+    {
+        "test_name": "Torrance Black Bear Diner",
+        "test_type": "Small diner in LA County",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3517050, 33.8060869],
+                "type": "Point"
+            }
+        },
+        "output": ["diners", "breakfast_brunch", "burgers"]
+    },
+
+    {
+        "test_name": "Inglewood In-N-Out Burger",
+        "test_type": "Chain fast food restaurant/drive thru off California freeway",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3324728, 33.9456953],
+                "type": "Point"
+            }
+        },
+        "output": ["burgers", "hotdogs"]
+    },
+
+    {
+        "test_name": "Torrance Memorial Medical Center",
+        "test_type": "Southern California hospital and medical center",
+        "input": {
+            "loc": {
+                "coordinates": [-118.3468133, 33.8124394],
+                "type": "Point"
+            }
+        },
+        "output": []
```

Comment:
does the medical center not have any matches? I bet it does. (at least, all of @valin's med center examples did)

Reply:
Both are fixed now. Apologies, earlier the non-business places weren't displaying categories on my computer. I can see them now